### ISSUE TITLE
NVDA reads items in showQuickPick menu as "blank"

### ISSUE THREAD

**Issue by cait1inhennessy** [Microsoft/vscode#58096]
Repro Steps:

1. Show any dropdown with the vscode.window.showQuickPick API
2. Start NVDA
3. Tab through the items in the dropdown menu and check how NVDA is reading them

OR
1. Install Live Share extension for VSCode
2. Press customized keyboard shortcut for "Sign In using browser(Sign In)" to open the Sign In window in browser.
3. Navigate to the name of the user logged in and press enter
4. Start NVDA
5. Tab to "Start Collaboration Session" in the quick pick menu
6. Check how NVDA reads this item

Actual Result:-
NVDA is reading the word "blank" for each item

Expected Result:
NVDA should read all the controls in the application correctly

User Impact:
Disabled users will face difficulties while using the application if Screen Reader doesn't read the content correctly

MAS Reference: 
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Note: In Mac, similar issue is observed, where Voice Over is not announcing anything for the above mentioned controls.

OS: Windows 10 Enterprise 17134.228

**Comment by cait1inhennessy** [comment]
![capture2](https://user-images.githubusercontent.com/16658549/45172593-a0867580-b1ba-11e8-9ee6-94ee07b15afe.PNG)

Here is our quick pick menu.  All the items - "share terminal", "share server", etc. - are read as "blank".

**Comment by chrmarti** [comment]
The issue with VoiceOver is tracked as upstream issue https://github.com/Microsoft/vscode/issues/52383.

**Comment by chrmarti** [comment]
It may read 'blank' for the empty input box, but it doesn't read 'blank' for each item in my tests with todays Insiders build. It does read the octicons, I can fix that as part of this issue.
